logstash-core/build.gradle

```diff
@@ -109,12 +109,14 @@ dependencies {
     compile 'org.apache.logging.log4j:log4j-core:2.6.2'
     compile 'com.fasterxml.jackson.core:jackson-core:2.7.4'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.7.4'
```

Comment:
I believe this will be a conflict when master gets reverted to 2.7.3 due to #7614